src/pages/Categories.js

```diff
@@ -0,0 +1,21 @@
+import AtlanticTeams from '../components/AtlanticTeams';
+import CentralTeams from '../components/CentralTeams';
+import NorthWestTeams from '../components/NorthWestTeams';
+import PacificTeams from '../components/PacificTeams';
+import SouthEastTeams from '../components/SouthEastTeams';
+import SouthWestTeams from '../components/SouthWestTeams';
+
+const Categories = () => { // eslint-disable-line
+  return (
+    <div className="category">
+      <AtlanticTeams />
+      <CentralTeams />
+      <NorthWestTeams />
+      <PacificTeams />
+      <SouthEastTeams />
+      <SouthWestTeams />
+    </div>
```

Comment:
- [ ] Great job making use of the default theme (pinkish) theme for your project. There is something small remaining. In the provided Behance design, the individual cards (NBA cards here) are in alternating styling. The first being either darker shade of pin and the second being a lighter shade of pink.

**This is how your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/203928547-da6e385c-79b3-462f-8d60-1b0683dfe79a.png)

**This is the design in the project requirements.**

![image](https://user-images.githubusercontent.com/86318284/203928808-deb393ab-41f3-4c80-8288-3a2f6ebad04f.png)

**Note the subtle difference? ** Kindly make this change to meet the design requirements.

Reply:
- [ ] This was not addressed